volttrontesting/platform/test_certs.py

```diff
@@ -66,15 +68,43 @@ def temp_volttron_home(request):
     """
     dirpath = tempfile.mkdtemp()
     os.environ['VOLTTRON_HOME'] = dirpath
+    debug_flag = os.environ.get('DEBUG', True)
```

Comment:
Probably should change this method to use the https://github.com/VOLTTRON/volttron/blob/be8668c64d16522a8f6da95924e4f3a14fd03ba7/volttrontesting/utils/platformwrapper.py#L181